relay-server/src/endpoints/security_report.rs

```diff
@@ -59,9 +59,11 @@ impl SecurityReportParams {
 
         if let Ok(items) = variant {
             for item in items {
-                let report_item =
-                    Self::create_security_item(&query, Bytes::from(item.to_owned().to_string()));
-                envelope.add_item(report_item);
+                let data = Bytes::from(item.to_owned().to_string());
+                if is_supported_type(&data) {
```

Comment:
Instead of parsing the data here, couldn't we filter out the unsupported type in the envelope processor (after or during the parsing step)?

Reply:
I first thought to do it in processor, but then , I've got an idea to discard those items as earlier as possible and later we won't do extra work.  But yeah, I will move this back to processor.

Reply:
done in [a0e3a4a](https://github.com/getsentry/relay/pull/3295/commits/a0e3a4aa596985b901e6c5980a605607c7e40d9e)